HelpSource/Classes/UnitTest.schelp

```diff
@@ -107,63 +107,92 @@ YourClass.test
 ::
 Runs the test class for code::YourClass::, which is assumed to be named code::TestYourClass::.
 
-If no test class if found it politely declines.
+If no test class is found, it politely declines.
 
 note::
-UnitTests for the Common library classes are kept in the CommonTests quark. 	This enables you to easily install and uninstall these tests.
+UnitTests for the Common library classes are kept in the CommonTests quark. This enables you to easily install and uninstall these tests.
 ::
 
 
 Make sure to implement the methods code::setUp:: and code::tearDown::. They will be called before and after the test.
 
 METHOD:: assert
 
+Asserts that an expression must be true.
+
 argument:: test
-Make sure that this returns a boolean, where code::true:: means that the test is passed.
+An expression which is expected to return a boolean, where code::true:: means that the test is passed.
 
 argument:: message
-posted if report is true.
+A message reflecting the result of the test.
+
+argument:: report
+Reports the result of the test if code::true:: (default is code::true::)
 
 argument:: onFailure
 If not code::nil::, a failure stops the tests and evaluates this function.
 
 code::
 UnitTest.new.assert(2 == 2, "Two does equal two.");
-UnitTest.new.assert(2 == 2.00001, "Two does equal two.");
+UnitTest.new.assert(2 == 2.00001, "Two doesn't equal 2.00001.");
 ::
 
-METHOD:: assertFloatEquals
-
-Make sure that the two floats code::a:: and code::b:: equal within a given range (code::within::).
+METHOD:: assertEquals
 
+Asserts that an expression code::got:: is equal to the value code::expected::.
```

Comment:
It should definitely be stated here that "equality" is determined according to the first argument's implementation of `==`.